src/app/containers/Header/ScriptLink/index.test.jsx

```diff
@@ -205,4 +248,28 @@ describe(`Script Link`, () => {
       ).toEqual('/foo/bar/cyr');
     });
   });
+
+  it('should not render when scriptLink toggle is off', () => {
```

Comment:
Great - could you also add a test to make sure the cookie isn't set when the variant cookie toggle is off, even though we have rendered the script link? So the same test as we currently have for `should set preferred variant cookie when ScriptLink is clicked`, but instead of expecting the function to have been called once, we expect it not to have been called.

Reply:
This is done now:)